### PR TITLE
Obfuscate results returned by listing GET endpoint

### PR DESCRIPTION
**Pull request by rekibnikufesin**

### CODE REVIEW

File: api/v1/listing.py [hunk 1]
```diff
@@ -47,7 +48,9 @@ def get(self):
         if response == 'Error retrieving record':
             return {'message': 'Error retrieving record'}, 500
         else:
-            return response, 200
+            print(response)
```
Comment (robrobbins) [inline api/v1/listing.py]: leaving this in on purpose?

File: api/v1/listing.py [hunk 2]
```diff
@@ -137,3 +140,67 @@ def delete(self):
...
+        sum_velocity = 0
+        velocity_points = 0
+        start_timestamp = datetime.now()
+        end_timestamp = datetime(1970, 1, 1, 12, 0, 0)
+        start_location = None
+        end_location = None
+        for location in listing['data']:
+            if 'velocity' in location:
+                sum_velocity = sum_velocity + float(location['velocity'])
+                velocity_points = velocity_points + 1
```
Comment (robrobbins) [inline api/v1/listing.py]: `velocity_points += 1` 
no?

File: api/v1/listing.py [hunk 3]
```diff
@@ -137,3 +140,67 @@ def delete(self):
...
+        obfuscated['num_data_points'] = len(listing['data'])
+        sum_velocity = 0
+        velocity_points = 0
+        start_timestamp = datetime.now()
+        end_timestamp = datetime(1970, 1, 1, 12, 0, 0)
+        start_location = None
+        end_location = None
+        for location in listing['data']:
+            if 'velocity' in location:
+                sum_velocity = sum_velocity + float(location['velocity'])
```
Comment (robrobbins) [inline api/v1/listing.py]: can we not use `+=` on these 2 vars? guess it doesn't really matter...

**Comment by robrobbins** [comment]
I'm fine with the method, just that one style nag. 

biggest thing is having Caroline maybe "sign off" on the shape of the "meta data"

File: api/v1/listing.py (hunk 1 above)
Comment (rekibnikufesin) [reply api/v1/listing.py]: lol, nope

File: api/v1/listing.py (hunk 3 above)
Comment (rekibnikufesin) [reply api/v1/listing.py]: Yeah. Updated. `+=` is cleaner.

**Pull request merged**